azure-communication-ui/azure-communication-ui/src/calling/java/com/azure/android/communication/ui/calling/service/sdk/CallingSDKEventHandler.kt

```diff
@@ -292,6 +315,14 @@ internal class CallingSDKEventHandler(
         mutedChangedListenersMap[id] = addOnIsMutedChangedEvent
         addedParticipant.addOnIsMutedChangedListener(mutedChangedListenersMap[id])
 
+        val addOnIsStateChangedEvent =
```

Comment:
Interesting! I haven't added this event listener in my iOS PR. But it still work. The onHold state would be updated correctly. 
Will dig more to see why iOS is working now without the state. And will add this listener if there is no other duplicated implementation